analyzer/codechecker_analyzer/analyzers/analyzer_types.py

```diff
@@ -111,22 +113,35 @@ def is_ignore_conflict_supported():
 def print_unsupported_analyzers(errored):
     """ Print error messages which occured during analyzer detection. """
     for analyzer_binary, reason in errored:
-        LOG.warning("Analyzer '%s' is enabled but CodeChecker is failed to "
-                    "execute analysis with it: '%s'. Please check your "
-                    "'PATH' environment variable and the "
-                    "'config/package_layout.json' file!",
-                    analyzer_binary, reason)
+        LOG.error("Analyzer '%s' is enabled but CodeChecker is failed to "
```

Comment:
I don't like this error message because it does not tell me where it found the analyzer binary.

Consider this error message instead (we dont need the "Please check your PATH..." part) :
CodeChecker analyze . --analyzers clangsa==13.0.1 -o reports
[ERROR 2023-10-11 20:09] - clangsa analyzer with the required version "13.0.1" not found. clangsa analyzer with version "13.0" found instead at /app/vbuild/UBUNTU18-x86_64/clang/13.0.0/bin/clang
[ERROR 2023-10-11 20:09] - Failed to run command because one or more given analyzers **could not** be found on your machine